src/seedu/addressbook/AddressBook.java

```diff
@@ -490,14 +490,19 @@ private static String getMessageForPersonsDisplayedSummary(ArrayList<String[]> p
     private static ArrayList<String[]> getPersonsWithNameContainingAnyKeyword(Collection<String> keywords) {
         final ArrayList<String[]> matchedPersons = new ArrayList<>();
         for (String[] person : getAllPersonsInAddressBook()) {
-            final Set<String> wordsInName = new HashSet<>(splitByWhitespace(getNameFromPerson(person)));
-            if (!Collections.disjoint(wordsInName, keywords)) {
+        	final String personName = getNameFromPerson(person);
+            final Set<String> wordsInName = new HashSet<>(splitByWhitespace(personName));
+            if (!isPersonWithNameContainingAnyKeyword(keywords, wordsInName)) {
                 matchedPersons.add(person);
             }
         }
         return matchedPersons;
     }
 
+	private static boolean isPersonWithNameContainingAnyKeyword(Collection<String> keywords, Set<String> wordsInName) {
```

Comment:
Missing header comment. All non trivial methods should have javadoc format header comments.